VAMobile/src/screens/HealthScreen/SecureMessaging/CancelConfirmations/ComposeCancelConfirmation.test.tsx

```diff
@@ -48,6 +48,24 @@ jest.mock('store/slices', () => {
         payload: '',
       }
     }),
+    resetSendMessageFailed: jest.fn(() => {
+      return {
+        type: '',
+        payload: '',
+      }
+    }),
+    resetSaveDraftFailed: jest.fn(() => {
+      return {
+        type: '',
+        payload: '',
```

Comment:
Had to update this so the unit tests worked once I deleted the claims and appeals slice. no idea why it was working before deletion, but whatever lol